model/model.go

```diff
@@ -312,6 +312,10 @@ type TableInfo struct {
 
 	// TiFlashReplica means the TiFlash replica info.
 	TiFlashReplica *TiFlashReplicaInfo `json:"tiflash_replica"`
+
+	// IsColumnar means the table is column-oriented.
+	// It's true when the engine of the table is TiFlash.
```

Comment:
Is TiFlash Only?

Reply:
Yes

Reply:
If so, I think it's better to say "It's true when the engine of the table is TiFlash only."